omod-1.11/pom.xml

```diff
@@ -110,7 +110,7 @@
 		<dependency>
 		      <groupId>org.apache.tomcat</groupId>
 		      <artifactId>jasper</artifactId>
-		      <version>6.0.18</version>
+		      <version>6.0.53</version>
```

Comment:
could this be the current version of this?

Reply:
@HerbertYiga [here](https://search.maven.org/artifact/org.apache.tomcat/jasper)  we might see that 6.0.53 i the last version and it    was release  in 2017